src/main/java/teammates/ui/controller/PublicImageServlet.java

```diff
@@ -2,11 +2,20 @@
 
 import java.io.IOException;
 
+
+import javax.servlet.http.HttpServlet;
```

Comment:
@hackosinn your build is currently failing because of the style checks

> :checkstyleMain[ant:checkstyle] [ERROR]
> /home/travis/build/TEAMMATES/teammates/src/main/java/teammates/ui/controller/PublicImageServlet.java:6: 'import' has more than 1 empty lines before. [EmptyLineSeparator]

> /home/travis/build/TEAMMATES/teammates/src/main/java/teammates/ui/controller/PublicImageServlet.java:6:	Avoid unused imports such as 'javax.servlet.http.HttpServlet'

> /home/travis/build/TEAMMATES/teammates/src/main/java/teammates/ui/controller/PublicImageServlet.java:13:	Avoid unused imports such as 'teammates.common.util.Assumption'

> /home/travis/build/TEAMMATES/teammates/src/main/java/teammates/ui/controller/PublicImageServlet.java:15:	Avoid unused imports such as 'teammates.common.util.HttpRequestHelper'

> /home/travis/build/TEAMMATES/teammates/src/main/java/teammates/ui/controller/PublicImageServlet.java:16:	Avoid unused imports such as 'teammates.common.util.Logger'